### DISCUSSION TITLE
The waiting list for Copilot chat link is broken.

### DISCUSSION THREAD

**Discussion by ELPIR** [community/community#68574] [Copilot Conversations] [answered]
Copilot chat link is broken.

Hi,

Does anyone know what happened to the Github Copilot chat waiting list link?
Since few days ago it's broken.

https://github.com/github-copilot/chat_waitlist_signup/join

**Comment by GhostOf0days** [comment, marked as answer]
This might be intended as it seems like it's in [public beta](https://docs.github.com/en/copilot/github-copilot-chat/about-github-copilot-chat) for those who should have it now.

**Comment by Akash1134** [comment]
👋 Hi @ELPIR , The above response from @GhostOf0days is correct, If you have a GitHub Copilot for Individuals subscription, you now have access to the GitHub Copilot Chat beta.

See our announcement from September 20: **[GitHub Copilot Chat beta now available for all individuals](https://github.blog/2023-09-20-github-copilot-chat-beta-now-available-for-all-individuals/)**

**Reply by CledsonBAR** [reply]
And is there a possibility of renewal for students? I had access until April 2023, due to the course having only theoretical subjects in the semester, I ended up not renewing it, yesterday I managed to renew the access, but it asks me to subscribe for a fee 

![git](https://github.com/community/community/assets/102316845/79ba842d-d88e-4b9b-983a-871c57a63708)
